.github/workflows/ci.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
           ~/go/pkg/mod
           ~/go/bin/
           ~/.kubebuilder/bin/k8s
-        key: gocache
+        key: go-cache-${{ hashFiles('**/go.sum') }}
```

Comment:
how does this make it faster?  Wouldn't the `gocache` get the same amount of hits (or maybe more) than hashing the deps checksums?